test/WebJobs.Script.Tests/Rpc/TestLanguageWorkerChannel.cs

```diff
@@ -96,5 +100,11 @@ public void RaiseWorkerErrorWithCustomTimestamp(DateTime timestamp)
             Exception testEx = new Exception("Test Worker Error");
             _eventManager.Publish(new WorkerErrorEvent(_runtime, Id, testEx, timestamp));
         }
+
+        public Task DrainInvocationsAsync()
+        {
```

Comment:
Update constructor to take in a bool : drainInvocationsSlowly and use this here to test the scenario where invocations take longer then the timeout

Reply:
I think I want to stick with the top down approach as mentioned...